presentation-api/receiving-ua/support/PresentationConnection_onmessage_receiving-ua.html

```diff
@@ -63,8 +63,8 @@
 
     assert_equals(connection.binaryType, 'arraybuffer', 'the default value of binaryType is "arraybuffer"');
     const eventWatcher = new EventWatcher(t, connection, 'message');
-    return eventWatcher.wait_for('message').then(evt => {
-      checkEvent(evt);
```

Comment:
@tomoyukilabs, I don't think it's typo, we often use `evt` as abbreviation of `event`.

Reply:
Yes, it’s correct. However, the next line contains `assert_equals(event.data, ...)`, which causes an error. So, this PR is intended to fix this inconsistency.

Reply:
Oh, sorry, you are right.